cloud-aws-cloudformation/src/main/resources/definitions/cdp/aws-cdp-ranger-audit-s3-policy.json

```diff
@@ -10,6 +10,12 @@
       ],
       "Resource": "arn:aws:s3:::${STORAGE_LOCATION_BASE}/ranger/audit/*"
     },
+    {
+      "Sid": "AccessToDatalakeBackupRestoreDirectory",
```

Comment:
how we can ensure that we dont break customers if they dont have this permission  ?

Reply:
I've created a [ticket](https://jira.cloudera.com/browse/CDPSDX-2999) to update the documentation for backup/restore as well.